xikolo-common/Data/Helper/SyncHelper.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  SyncHelper.swift
+//  xikolo-ios
+//
+//  Created by Max Bothe on 15.01.18.
+//  Copyright © 2018 HPI. All rights reserved.
+//
+
+import Foundation
+import BrightFutures
+import CoreData
+
+struct SyncHelper {
+
+    static private let dateFormatter: DateFormatter = {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "EEEE, dd LLL yyyy HH:mm:ss zzz"
+        return formatter
+    }()
+
+    static func syncResources<Resource>(withFetchRequest fetchRequest: NSFetchRequest<Resource>,
+                                        withQuery query: MultipleResourcesQuery<Resource>,
+                                        deleteNotExistingResources: Bool = true) -> Future<SyncEngine.SyncMultipleResult, XikoloError> where Resource: NSManagedObject & Pullable {
+        return SyncEngine.syncResources(withFetchRequest: fetchRequest, withQuery: query, deleteNotExistingResources: deleteNotExistingResources).onSuccess { syncResult in
+            print("Verbose: Successfully merged resources of type: \(Resource.type)")
```

Comment:
Aren’t we using a real logger? Or is this the way to go? If so I would like a wrapper function ...

Reply:
I'm on it. See #367.